src/main/cpp/blaze.cc

```diff
@@ -1423,14 +1423,28 @@ static map<string, EnvVarValue> PrepareEnvironmentForJvm() {
   if (!blaze::GetEnv("http_proxy").empty() ||
       !blaze::GetEnv("HTTP_PROXY").empty()) {
     BAZEL_LOG(WARNING)
-        << "detected http_proxy set in env, setting no_proxy for localhost.";
+        << "detected HTTP_PROXY set in env, setting NO_PROXY for localhost.";
```

Comment:
http_proxy (and no_proxy) are the more common forms, so the preference in this pull request for the uppercase forms (HTTP_PROXY and NO_PROXY) seems like the wrong thing to do...

Maybe:

"detected http_proxy/HTTP_PROXY set in env -- will set no_proxy/NO_PROXY for localhost..."